src/Component/Configuration/Loader/BabelLoader.php

```diff
@@ -32,7 +32,9 @@ public static function applyConfiguration(NodeBuilder $node_builder)
                     ->arrayNode('presets')
```

Comment:
I still don't know if Babel still transpiles my ES2015 scripts automatically if I don't explicitly tell it to use the 'es2015' preset. Can you answer me this question?

Does Babel have a "default" preset if nothing is specified?

If not, like I asked before, please add a default here to keep this PR backwards compatible with previous versions.